src/pages/home/components/Header/Header.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react';
+
+import Header from './Header';
+
+describe('when initialize', () => {
+  it('renders without crashing', () => {
```

Comment:
Dá pra tentar uma ousadia aqui nesse teste.

Testar que eles existem é OK.

Mas quando clica neles, o que acontece? Vale verificar o que acontece.

Eu seguiria pra testar isso em um caminho tipo esse, mas isso é só uma sugestão.

`const spy = jest.spyOn(window.location, 'assign').mockImplementation(() => {});`

Reply:
Eu concordo. 
Porém, to tendo um problema ao tentar usar algum método do jest no arquivo de testes:
![image](https://github.com/devhatt/octopost/assets/117240363/dddf6990-1ed3-4c72-a72e-67eaaa29a2c0)
Mesmo importando as bibliotecas de teste: 
![image](https://github.com/devhatt/octopost/assets/117240363/3339c4c8-4e96-400b-92a8-b58fabe1213c)
Tem algum conselho do que posso fazer?

Reply:
Perdão! Eu sugeri jest, mas no projeto usamos vitest. Então use o vi.spyOn